### ISSUE TITLE
Resource usage has increased by 6%

### ISSUE THREAD

**Issue by xobs** [im-tomu/valentyusb#12]
With the latest CDC patch, resource utilization has increased by 6%. As a result, valentyusb no longer fits on Fomu:

```
Info: Device utilisation:
Info:            ICESTORM_LC:  5567/ 5280   105%
Info:           ICESTORM_RAM:    30/   30   100%
Info:                  SB_IO:    14/   96    14%
Info:                  SB_GB:     8/    8   100%
Info:           ICESTORM_PLL:     1/    1   100%
Info:            SB_WARMBOOT:     1/    1   100%
Info:           ICESTORM_DSP:     0/    8     0%
Info:         ICESTORM_HFOSC:     0/    1     0%
Info:         ICESTORM_LFOSC:     0/    1     0%
Info:                 SB_I2C:     0/    2     0%
Info:                 SB_SPI:     0/    2     0%
Info:                 IO_I3C:     0/    2     0%
Info:            SB_LEDDA_IP:     1/    1   100%
Info:            SB_RGBA_DRV:     1/    1   100%
Info:         ICESTORM_SPRAM:     4/    4   100%
```

**Comment by xobs** [comment]
cc @bunnie

**Comment by bunnie** [comment]
Gotcha. So I should change the code so that it cuts out the CDC elements, as much as possible, if you're not using it?

**Comment by bunnie** [comment]
(to wit, the CDC stuff by right infers more registers because more synchronization, and it also duplicates computation in clk12 and sysclk domain for very fast paths that can't afford the synchronization overhead)

**Comment by bunnie** [comment]
PR #13 should resolve this.

**Comment by xobs** [comment]
This is solved by PR #13. Thanks!
